src/util/wallet-utils/src/sign-psbt.ts

```diff
@@ -34,7 +32,9 @@ export async function signPSBTXverse({ hex, address }: SignPSBTParams) {
     throw new Error(`Error signing PSBT`);
   }
 
-  return bytesToHex(
-    Uint8Array.from(atob(response.result.psbt), (c) => c.charCodeAt(0)),
-  );
+  return base64ToHex(response.result.psbt);
+}
+
+export function base64ToHex(base64: string) {
+  return bytesToHex(Uint8Array.from(atob(base64), (c) => c.charCodeAt(0)));
```

Comment:
nit: there's also `Uint8Array.fromBase64` (which they say it's preferred over `atob`)

Reply:
I would have loved to use this as a native API however its support is very very limited unfortunately 
<img width="784" alt="image" src="https://github.com/user-attachments/assets/c34ff55c-f53e-4879-ab2b-2bdb2f984dec" />